README.md

```diff
@@ -51,3 +51,111 @@ Install an OpenPGP implementation:
   to a package called `gnupg2`
 * Android: https://www.openkeychain.org/
 * iOS: https://privacyapp.io/
+
+## After the event
+
+You should now have a table with a set of validated fingerprints and ids. You
+can use this to now sign any of those identities that you have personally
+validated.
+
+This guide is written assuming you have `gpg2` installed as `gpg` if any of
```

Comment:
These instructions work on which platform?

Reply:
any with a terminal a `sh` like shell and gpg 2+ installed,

Reply:
I think you should explicitly say what ios/android/windows should use